package/gluon-scheduled-domain-switch/Makefile

```diff
@@ -0,0 +1,13 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-scheduled-domain-switch
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-scheduled-domain-switch
+  TITLE:=Allows scheduled migrations between domains
+  DEPENDS:=+gluon-core
```

Comment:
I found another thing... Please add `@GLUON_MULTIDOMAIN` to DEPENDS, so the package can only be selected in multi-domain configurations.

Reply:
Fixed